modules/imgproc/src/contours_new.cpp

```diff
@@ -27,69 +27,77 @@ struct Trait
 };
 
 static const schar MASK8_RIGHT = '\x80'; // 1000 0000
-static const schar MASK8_NEW   = '\x02'; // 0000 0010 (+2)
+static const schar MASK8_NEW = '\x02'; // 0000 0010 (+2)
 static const schar MASK8_FLAGS = '\xFE'; // 1111 1110 (-2)
 static const schar MASK8_BLACK = '\x01'; // 0000 0001 - black pixel
 
-static const schar MASK8_LVAL  = '\x7F'; // 0111 1111 (for table)
+static const schar MASK8_LVAL = '\x7F'; // 0111 1111 (for table)
 
-template<>
+template <>
 struct Trait<schar>
 {
-    static inline bool checkValue(const schar * elem, const schar *) {
+    static inline bool checkValue(const schar* elem, const schar*)
+    {
         return *elem != 0;
     }
-    static inline bool isVal(const schar * elem, const schar *) {
+    static inline bool isVal(const schar* elem, const schar*)
+    {
         return *elem == MASK8_BLACK;
     }
-    static inline bool isRight(const schar * elem, const schar *) {
+    static inline bool isRight(const schar* elem, const schar*)
+    {
         return (*elem & MASK8_RIGHT) != 0;
     }
-    static inline void setRightFlag(schar * elem, const schar *, schar nbd) {
+    static inline void setRightFlag(schar* elem, const schar*, schar nbd)
+    {
         *elem = nbd | MASK8_RIGHT;
     }
-    static inline void setNewFlag(schar * elem, const schar *, schar nbd) {
+    static inline void setNewFlag(schar* elem, const schar*, schar nbd)
+    {
         *elem = nbd;
     }
 };
 
 static const int MASK_RIGHT = 0x80000000; // 100..000
-static const int MASK_NEW   = 0x40000000; // 010..000
+static const int MASK_NEW = 0x40000000; // 010..000
 static const int MASK_FLAGS = 0xC0000000; // right + new
-static const int MASK_VAL   = 0x3FFFFFFF; // ~flags - pixel label
+static const int MASK_VAL = 0x3FFFFFFF; // ~flags - pixel label
```

Comment:
Is there clang-format option 'do not touch'?

Reply:
We can enclosure a block of code between `// clang-format on` - `// clang-format off` comments to ignore it.

But I believe in general it is better to avoid alignment of values or comments in such blocks to simplify future merges. Thus I chose to disable these rules: `AlignConsecutiveAssignments: None` and other `AlignConsecutive*` options (see https://releases.llvm.org/12.0.0/tools/clang/docs/ClangFormatStyleOptions.html for details)